tests/src/main/java/com/linkedin/kmf/tests/BasicEndToEndTest.java

```diff
@@ -83,133 +84,151 @@ public void awaitShutdown() {
+      .type(String.class)
+      .metavar("REPORT_INTERVAL_SEC")
+      .dest("reportIntervalSec")
+      .help("Interval in sec with which to export stats");
 
     parser.addArgument("--record-delay-ms")
-        .action(store())
-        .required(false)
-        .type(String.class)
-        .metavar("RECORD_DELAY_MS")
-        .dest("recordDelayMs")
-        .help("The delay in ms before sending next record to the same partition");
+      .action(store())
+      .required(false)
+      .type(String.class)
+      .metavar("RECORD_DELAY_MS")
+      .dest("recordDelayMs")
+      .help("The delay in ms before sending next record to the same partition");
 
     parser.addArgument("--latency-percentile-max-ms")
-        .action(store())
-        .required(false)
-        .type(String.class)
-        .metavar("LATENCY_PERCENTILE_MAX_MS")
-        .dest("latencyPercentileMaxMs")
-        .help("The maximum value in ms expected for latency percentile metric. "
-            + "The percentile will be reported as Double.POSITIVE_INFINITY if its value exceeds the max value.");
+      .action(store())
+      .required(false)
+      .type(String.class)
+      .metavar("LATENCY_PERCENTILE_MAX_MS")
+      .dest("latencyPercentileMaxMs")
+      .help("The maximum value in ms expected for latency percentile metric. " +
+            "The percentile will be reported as Double.POSITIVE_INFINITY if its value exceeds the max value.");
 
     parser.addArgument("--latency-percentile-granularity-ms")
-        .action(store())
-        .required(false)
-        .type(String.class)
-        .metavar("LATENCY_PERCENTILE_GRANULARITY_MS")
-        .dest("latencyPercentileGranularityMs")
-        .help(
-            "The granularity in ms of latency percentile metric. This is the width of the bucket used in percentile calculation.");
+      .action(store())
+      .required(false)
+      .type(String.class)
+      .metavar("LATENCY_PERCENTILE_GRANULARITY_MS")
+      .dest("latencyPercentileGranularityMs")
+      .help("The granularity in ms of latency percentile metric. This is the width of the bucket used in percentile calculation.");
 
     parser.addArgument("--auto-topic-creation")
-        .action(store())
-        .required(false)
-        .type(Boolean.class)
-        .metavar("AUTO_TOPIC_CREATION_ENABLED")
-        .dest("autoTopicCreationEnabled")
-        .help(ProduceServiceConfig.AUTO_TOPIC_CREATION_ENABLED_DOC);
+      .action(store())
+      .required(false)
+      .type(Boolean.class)
+      .metavar("AUTO_TOPIC_CREATION_ENABLED")
+      .dest("autoTopicCreationEnabled")
+      .help(ProduceServiceConfig.AUTO_TOPIC_CREATION_ENABLED_DOC);
+
+    parser.addArgument("--topic-rebalance-enabled")
+      .action(store())
+      .required(false)
+      .type(Boolean.class)
+      .metavar("REBALANCE_ENABLED")
+      .dest("rebalanceEnabled")
+      .help("When true this creates and starts a topic rebalancer.");
+
+    parser.addArgument("--topic-rebalance-ms")
```

Comment:
I am not sure "topic-rebalance-ms" tells user what is means. How about "topic-rebalance-interval-ms"?